src/tables/cff.js

```diff
@@ -919,7 +919,8 @@ function makeStringIndex(strings) {
     ]);
     t.strings = [];
     for (var i = 0; i < strings.length; i += 1) {
-        t.strings.push({name: 'string_' + i, type: 'STRING', value: strings[i]});
+        if (strings[i])
+            t.strings.push({name: 'string_' + i, type: 'STRING', value: strings[i]});
     }
```

Comment:
This fixes the symptom, but not necessarily the problem: why is there an empty string? Because if there's one thing CFF doesn't allow, it's empty strings in the string index, so it feels more like something went wrong with the input getting sent into `makeStringIndex` (e.g. somehow the charset or dict parses are generating illegal strings that aren't getting caught early enough)